client/components/productOption.js

```diff
@@ -20,15 +21,17 @@ class ProductOption extends React.Component {
   render() {
     return (
       <div className="productOption">
-        <label>{this.props.optionName}</label><br/>
-        <select  value={this.state.optionSelection} onChange={this.handleDropDownSelection} >
-          <option value={''}>Select an option</option>
-          {this.props.choices.map((choice, idx) => {
-            return (
-              <option value={`${choice.choice} $${choice.adjustedPrice || ''}`} key={idx}>
-                {`${choice.choice}  `}  {choice.adjustedPrice ? `($${choice.adjustedPrice})` : ''}
-              </option>
-            )
+        <div className={styles.optionLabel}>
+          <label className={styles.optionText}>{this.props.optionName}</label>
+        </div>
+        <select className={styles.optionSelect} value={this.state.optionSelection} onChange={this.handleDropDownSelection} >
+        <option value={''}>Select an option</option>
+        {this.props.choices.map((choice, idx) => {
+          return (
+            <option value={`${choice.choice} $${choice.adjustedPrice || ''}`} key={idx}>

```

Comment:
I'm not 100% on what `choice.choice` is here, but your key can be `choice.choice` (or whatever other unique identifier); it'd be a better choice than `idx`.